crates/ide-assists/src/handlers/fix_visibility.rs

```diff
@@ -7,32 +7,31 @@ use syntax::{
 
 use crate::{AssistContext, AssistId, Assists};
 
-// FIXME: this really should be a fix for diagnostic, rather than an assist.
-
-// Assist: fix_visibility
-//
-// Makes inaccessible item public.
-//
-// ```
-// mod m {
-//     fn frobnicate() {}
-// }
-// fn main() {
-//     m::frobnicate$0();
-// }
-// ```
-// ->
-// ```
-// mod m {
-//     $0pub(crate) fn frobnicate() {}
-// }
-// fn main() {
-//     m::frobnicate();
-// }
-// ```
+/// Assist: fix_visibility
```

Comment:
or would you rather this be in a separate PR?

Reply:
Please undo the comment conversion. This is intentionally not a doc comment (its used by our tooling to extract as a document into the manual)